docs/configuration.md

```diff
@@ -0,0 +1,27 @@
+## Configuration
+Visibility plugin configuration parameters are provided whenever a player instance is created.
+```js
+var config = {
+  plugins: {
+    visibility: {
+      // Visibility configuration here
+    }
+  }
+};
+var player = KalturaPlayer.setup(config);
+```
+
+#### Configuration Structure
+
+The configuration uses the following structure:
+
+```js
+{
+}
+```
+
+#### Default Configuration Values
+```js
```

Comment:
?

Reply:
I didn't do any documentation yet. Will complete this in the next phase